src/calculation/ClauseResultsBuilder.ts

```diff
@@ -736,7 +747,8 @@ export function buildPopulationRelevanceMap(
 
   // If DENEX is true then NUMER, NUMEX and DENEXCEP not calculated
   if (hasResult(PopulationType.DENEX, results) && getResult(PopulationType.DENEX, results) === true) {
-    if (!(group && MeasureBundleHelpers.hasMultipleIPPs(group))) {
+    // Do not apply this to ratio measures (numerator and denominator are independent from each other)
+    if (!MeasureBundleHelpers.isRatioMeasure(group, measureScoringCode)) {
```

Comment:
I think there's one more case of these that is missing (down on line 772).

There is a block of code that falses out the DENEXCEP if numer is true, but we should not do this for ratio measures according to the Venn D since DENEXCEP is a subset of the denom which is independent from the numer

Reply:
Dang these were the two changes I was going to request hahah